src/Controllers/SchemaController.js

```diff
@@ -1405,6 +1441,8 @@ export default class SchemaController {
       }
       // requiresAuthentication passed, just move forward
       // probably would be wise at some point to rename to 'authenticatedUser'
```

Comment:
Stumbled upon this old comment I though `authenticated` is a better description of a group of users than `requiresAuthentication`, just left the former variant for backwards compatibility. I'd propose to replace it in all the docs and may be drop completely by next major release or whenever. This also can be applied to CLP.

Reply:
it may be a better name but I am not sure if it is worth to be done since it is a breaking change.

Reply:
Ok just realized there is no point in supporting `requiresAuthentication` for Protected Fields at all. Though for some reason it could pass validation regex before (used same regex as for CLP permissions) - it had no real effect anyway. So I'll only leave `authenticated` for Protected Fields.